### ISSUE TITLE
Use stable tailwind flag for remix

### ISSUE THREAD

**Issue by kinggoesgaming** [tailwindlabs/tailwindcss.com#1595]
As of remix `1.16.0`, `future.unstable_tailwind` in `remix.config.js` is deprecated. Instead the stable flag `tailwind` is used.

In essence the change would be as follows for https://tailwindcss.com/docs/guides/remix (Step 2):

```diff
/** @type {import('@remix-run/dev').AppConfig} */
module.exports = {
  future: {
-    unstable_tailwind: true,
  },
+  tailwind: true,
}
```

**Comment by thecrypticace** [comment]
I've merged #1569 that updates the guide for this. It should be deployed within the next few minutes. Appreciate it!

**Comment by kinggoesgaming** [comment]
Doing my part :)
